### DISCUSSION TITLE
Guidance on getting Playnite fullscreen to open on virtual display

### DISCUSSION THREAD

**Discussion by epicmeatwad** [ClassicOldSong/Apollo#166] [General]
I cant for the life of me figure out how to make playnite open on the virtual display.   I have 'Always Use Virtual ticked' and 'Enforce Virtual Display Primary' ticked in the application settings for Apollo.   I'll get the playnite splash screen on the virtual display that my tablet is using and then playnite goes full screen on my main physical monitor. 

Thanks for any help!

**Comment by epicmeatwad** [comment]
Got it figured out, when Playnite is in fullscreen mode, you can set the target display destination, so I switched it to Primary.    Apollo works great, loving it compared to all the voodoo I had to do with Sunshine.

**Reply by ClassicOldSong** [reply]
Glad you get it working!
